### PR TITLE
add lon-lat orientation + shorten 'waypoint.waypoint_type'

### LINKED ISSUE #286: Waypoint editing form
* [x] Don't show (and don't get) associated outings (see for instance http://c2corgv6-demo.gis.internal:6555/waypoints/edit/735236/en)

### PR DESCRIPTION
**Pull request by ginold**
instead of writing `waypoint.waypoint_type` I init `{{type = waypoint.waypoint_type}}` which is more concise. I replaced all of the occurrences of `waypoint.waypoint_type`.

Longitude and latitude use °E and °N.
fixes #286

### CODE REVIEW

File: c2corg_ui/templates/waypoint/edit.html [hunk 1]
```diff
@@ -174,7 +174,7 @@ <h2 class="heading show-phone"><span translate>geolocation</span></h2>
               <label><span translate>longitude</span> *</label>
               <div class="input-group">
                 <input type="number" name="longitude" ng-model="waypoint.lonlat.longitude" ng-change="editCtrl.updateMap()" class="form-control" required required min="-90" max="90" />
-                <span class="input-group-addon">°</span>
+                <span class="input-group-addon">E°</span>
```
Comment (asaunier) [inline c2corg_ui/templates/waypoint/edit.html]: Rather ``°E`` and ``°N``
https://en.wikipedia.org/wiki/Geographic_coordinate_system#/media/File:This_is_a_basic_photo_showing_that_latitude_lines_are_horizontal_and_longitude_lines_are_vertical-_2014-07-25_20-36.jpg

File: c2corg_ui/templates/waypoint/edit.html [hunk 2]
```diff
@@ -463,7 +463,7 @@ <h2 class="heading show-phone" ><span translate>Numbers</span></h2>
             ## CUSTODIANSHIP
             <div class="form-group data half" ng-if="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'" ng-class="{'has-success': waypoint.custodianship}">
               <label><span translate>custodianship</span> <span ng-if="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'camp_site' || waypoint.waypoint_type == 'hut'">*</span></label>
-              <select class="form-control" ng-options="type as mainCtrl.translate(type) for type in ${custodianship_types}" ng-model="waypoint.custodianship"><option value=""></option></select>
+              <select class="form-control" name="custodianship" ng-options="type as mainCtrl.translate(type) for type in ${custodianship_types}" ng-model="waypoint.custodianship" ng-required="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'"><option value=""></option></select>
```
Comment (asaunier) [inline c2corg_ui/templates/waypoint/edit.html]: Connection with the coords? :P 
By the way isn't there a more concise way to write ``waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'camp_site'``?

Comment (ginold) [inline c2corg_ui/templates/waypoint/edit.html]: maybe by init a variable {{something = waypoint.waypoint_type}}

Comment (asaunier) [inline c2corg_ui/templates/waypoint/edit.html]: I meant something like ``['gite', 'hut', 'camp_site'].indexOf(waypoint.waypoint_type) != -1``.

**Comment by ginold** [comment]
Merge?

**Comment by asaunier** [comment]
:+1:

**Pull request merged**